pkg/cmd/workspace/delete.go

```diff
@@ -133,6 +134,7 @@ var DeleteCmd = &cobra.Command{
 func init() {
 	DeleteCmd.Flags().BoolVarP(&allFlag, "all", "a", false, "Delete all workspaces")
 	DeleteCmd.Flags().BoolVarP(&yesFlag, "yes", "y", false, "Confirm deletion without prompt")
+	DeleteCmd.Flags().BoolVarP(&forceFlag, "force", "f", false, "Delete a workspace by force")
 }
 
 func DeleteAllWorkspaces() error {
```

Comment:
The force flag should be passed to `DeleteAllWorkspaces` as well.

![Screenshot 2024-06-10 at 16 45 43](https://github.com/daytonaio/daytona/assets/26512078/5c837d66-b1e9-43cd-8889-199a50d32ec5)

(the second command from the screenshot should complete successfully)